modules/gdscript/gdscript_compiler.cpp

```diff
@@ -43,6 +43,10 @@ bool GDScriptCompiler::_is_class_member_property(CodeGen &codegen, const StringN
 		return false;
 	}
 
+	if (codegen.parameters.has(p_name)) {
+		return false; //shadowed
+	}
+
 	if (codegen.locals.has(p_name)) {
 		return false; //shadowed
 	}
```

Comment:
```suggestion
	if (codegen.parameters.has(p_name) || codegen.locals.has(p_name)) {
		return false; // Shadowed.
	}
```
maybe? Not sure what's the preferred style for GDScript specifically, up to @vnen to say I guess.

Reply:
This is indeed more concise :smile: